docs/developer/chart-repo.md

```diff
@@ -45,7 +45,7 @@ This builds the `chart-repo` Docker image. Please refer to [Monocular Developers
 ### Running in development
 
 ```bash
-export MONGO_PASSWORD=$(kubectl get secret --namespace kubeapps kubeapps-mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
+export MONGO_PASSWORD=$(kubectl get secret --namespace kubeapps kubeapps-mongodb -o go-template='{{index .data "mongodb-root-password" | base64decode}}')
```

Comment:
can you add here a ` && echo` (as in the last example). It's useful to separate the result.

Reply:
Does it really make sense here? Considering the output is not displayed

Example:
```bash
$ export MONGO_EG=$(kubectl get secret --namespace kubeapps kubeapps-mongodb -o go-template='{{index .data "mongodb-root-password" | base64decode}}')
$
```

with `&& echo` change
```bash
$ export MONGO_EG=$(kubectl get secret --namespace kubeapps kubeapps-mongodb -o go-template='{{index .data "mongodb-root-password" | base64decode}}') && echo

$
```

If this is the desired change, no problem, just want to confirm.

Reply:
you are correct, here it should not include the trailing `\n` (I just executed the `kubectl` part :sweat_smile: )

Reply:
Ok, I just made the change for the other command. Kept this one as it is! 😄